src/components/Banner.tsx

```diff
@@ -0,0 +1,9 @@
+interface Props {
+	children: React.ReactNode;
+}
+
+export const Banner = ({ children }: Props) => (
+	<article className="mb-8 border-l-4 border-yellow-400 bg-yellow-50 p-4">
```

Comment:
write TODO: to add other colors

Reply:
I'll also create an issue for that so some community member might possibly contribute as a `first good issue` 😉

Reply:
https://github.com/naugtur/meetjs.pl/issues/129